.travis.yml

```diff
@@ -20,17 +20,22 @@ matrix:
 sudo: false
 cache:
   cargo: true
-  directories:
-    - $HOME/elfutils
+before_script:
+  - (which cargo-install-update && cargo install-update cargo-update) || cargo install cargo-update
+  - (which rustfmt && cargo install-update rustfmt) || cargo install rustfmt
+  - (which cargo-prune && cargo install-update cargo-prune) || cargo install cargo-prune
 script:
-  - curl -sSLO https://github.com/maidsafe/QA/raw/master/Bash%20Scripts/Travis/build_and_run_tests.sh
-  - travis_wait 60 . build_and_run_tests.sh
-  - cd $TRAVIS_BUILD_DIR && travis_wait 20 cargo build --example key_value_store
-  - cd $TRAVIS_BUILD_DIR && travis_wait 20 cargo build --example ci_test
-  # - cd $TRAVIS_BUILD_DIR && cargo run --release --example ci_test > output.log
-  # - cat output.log
+  - if [ "${TRAVIS_RUST_VERSION}" = stable ]; then
+      (
+        set -x;
+        cargo fmt -- --write-mode=diff &&
+        cargo rustc --verbose -- --test -Zno-trans &&
+        cargo rustc --verbose --example ci_test -- -Zno-trans &&
+        cargo rustc --verbose --example key_value_store -- -Zno-trans &&
+        cargo rustc --verbose -- -Zno-trans &&
+        cargo test  --verbose --features use-mock-crust
```

Comment:
Yes, it does. I think the only thing it excludes is `tests/ci.rs`, and we haven't been running that on Travis before this change either.